website/content/docs/auth/azure.mdx

```diff
@@ -234,6 +234,63 @@ tool.
        https://127.0.0.1:8200/v1/auth/azure/role/dev-role
    ```
 
+## Root credential rotation
+
+If the mount is configured with credentials directly, the credential's key may be
+rotated to a Vault-generated value that is not accessible by the operator.
+This will ensure that only Vault is able to access the "root" user that Vault uses to
+manipulate dynamic & static credentials.
+
+```shell-session
+vault write -f auth/azure/rotate-root
+```
+
+### Schedule-based root credential rotation
+
+<EnterpriseAlert product="vault" />
+
+The Azure auth engine supports configuring schedule-based automatic
+credential rotation for the root credential with the
+[rotation_schedule](/vault/api-docs/auth/azure#rotation_schedule) field.
+For example:
```

Comment:
```suggestion
Use the [`rotation_schedule`](/vault/api-docs/auth/azure#rotation_schedule) field
to configure schedule-based, automatic credential rotation for root credentials in
the Azure auth engine. For example, the following command set the rotation to
occur every Saturday at midnight (00:00):
```
Style correction: write in active voice

Reply:
applied (with necessary plugin name and path edits) to all four plugins